### ISSUE TITLE
Enable support for splitscreen

### ISSUE THREAD

**Issue by Hedronmx** [c0dycode/BL3HotfixWebUI#13]
Would be awesome to play with splitscreen enabled using nucleus. Not sure how feasible this is... Thanks for this great tool

**Comment by c0dycode** [comment]
This has been requested before and will be added/supported in the next version.
Whenever I have some time I'm working on that update, but unfortunately I currently do not have that much time to spare to do so. So this might take a bit.

**Comment by Hedronmx** [comment]
Yeah! I understand. I saw your reply in one of the forums but I decided to open this up just to have it organized here
